docs/tutorials/tphi_characterization.rst

```diff
@@ -28,7 +32,7 @@ we compute the results for :math:`T_\varphi.`
     
     # Create a pure relaxation noise model for AerSimulator
     noise_model = NoiseModel.from_backend(
-        FakeVigo(), thermal_relaxation=True, gate_error=False, readout_error=False
+        FakeVigo(), thermal_relaxation=True, gate_error=True, readout_error=False
```

Comment:
Why change here?

Reply:
I thought it might be a more realistic model but turns out it doesn't matter, so I reverted the change.